src/quo2/components/banners/__tests__/banner_component_spec.cljs

```diff
@@ -15,3 +15,13 @@
                       (.toBeTruthy))
                   (-> (js/expect (rtl/screen.getByText "5"))
                       (.toBeTruthy))))
+
+(js/global.test "banner component fires an event when pressed"
```

Comment:
Shouldn't the tests be defined inside `banners/banner/` directory? If you do it, then test descriptions like `banner component <bla>` will be unnecessary, because it would be clear the component tests are only related to the banner, and nothing else.

Reply:
yeah sure we can do that. 
I was thinking just at the top level and each component has it's own spec/test file. But I can implement this suggestion as it's clearer of what component the test relates to and will help avoid confusion 👍